### PR TITLE
Ada-49

### PR DESCRIPTION
**Pull request by SnobbyDragon**
Counts the unique members (using dot number). To test:
1. Add students to the `student` table
2. Comment out the line that says to comment out in `acmw-db-app/postgres/pg-query.js`
3. Run `npm run dev`
4. Go to `http://localhost:3000/api/student/totalUniqueMembers`

Also, I don't know why my other commit for that other story card thing was there. Maybe I deliriously worked on the same branch...

### CODE REVIEW

File: acmw-db-app/pages/api/student/[pid].js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import pgQuery from '../../../postgres/pg-query.js';
+
+// GET /api/student/totalUniqueMembers
+async function totalUniqueMembers () {
```
Comment (jgeorge37) [inline acmw-db-app/pages/api/student/[pid].js:4]: needs to incorporate “for a given academic year” i.e. check that they attended a meeting in AU__ or SP__

**Review by jgeorge37 — CHANGES_REQUESTED** [review]

**Comment by SnobbyDragon** [comment]
UHHHHH my thing for the other branch is still there so please ignore that or tell me to get rid of it.

**Review by jgeorge37 — CHANGES_REQUESTED** [review]
Should not need to involve the student table if we are just counting student ids from the meeting_student table

**Comment by jgeorge37** [comment]
Please close and re-open this as a Dev PR rather than master PR per the slack message!

**Review by jgeorge37 — APPROVED** [review]

**Pull request merged**